core/tools.js

```diff
@@ -123,6 +123,42 @@
 	 * @singleton
 	 */
 	CKEDITOR.tools = {
+
+		/**
+		 * Verifies the safety of an html string and returns trusted version.
+		 *
+		 * @param {string} html The safe html string.
+		 * @param {string} justification A short justification for why this
+		 * html string can be verified as safe.
+		 * @returns {TrustedHTML | string} the same html string but as a
+		 * Trusted Type or a string if TT is not supported.
+		 */
+		htmlSafeByReview: function (html, justification) {
+			// Used for Trusted Types assertions 
```

Comment:
nit: This comment might be confusing for devs who aren't exactly familiar with TT. Maybe let's rephrase to highlight that this assertion checks that there's a non-empty justification